streamz/__init__.py

```diff
@@ -3,6 +3,7 @@
 from .core import *
 from .graph import *
 from .sources import *
+from .sinks import *
```

Comment:
I agree, but it comes down to backwards-compatibility. Since this has already been done like this (`*`), someone might import something internal directly from `streamz`, like `_global_sinks` for some custom implementation of a sink. We might say that it's OK that people will have to tweak code a little bit or call it 2.0, but I don't want to change it without your/other maintainers' "OK".

Reply:
There's another problem with this: docs are written on the assumption that everything is importable directly from `streamz`, so I'm leaving this as is for now.

Reply:
Fair. We may come back to this.